src/makeRobot.js

```diff
@@ -39,6 +39,66 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
if key and value have the same name you can use short js syntax for them
```suggestion
    name,
    wheels,
    version,
```